{"repository":"https://github.com/auhau/ipfs-publish","dockerfilePath":"Dockerfile","startTime":1672877760449,"endTime":1672877819141,"clone":{"startTime":1672877760449,"stdout":"","stderr":"Cloning into '/tmp/dinghy-analysis/auhau/ipfs-publish'...\n","error":null,"endTime":1672877761222,"commit":"9ee92f0d3f6fdebd1f51f8bcad244e6147b8b2b8"},"originalBuild":{"startTime":1672877761303,"endTime":1672877819057,"error":{"code":1,"killed":false,"signal":null,"cmd":"docker build --no-cache --force-rm -t ipfs-publish:latest -f /tmp/dinghy-analysis/auhau/ipfs-publish/Dockerfile ."},"stdout":"Sending build context to Docker daemon  236.5kB\r\r\nStep 1/15 : FROM python:3.7\n3.7: Pulling from library/python\n32de3c850997: Already exists\nfa1d4c8d85a4: Already exists\nc796299bbbdd: Already exists\n81283a9569ad: Already exists\n60b38700e7fb: Already exists\n0f67f32c26d3: Already exists\n24e64f2eb938: Already exists\nece0ca00f21c: Already exists\n9a5f30b220ac: Already exists\nDigest: sha256:bf85a74f4ace82f3503c2199aaae10e7a8e370bc7e42fd246c5774891f1fab0b\nStatus: Downloaded newer image for python:3.7\n ---> 1f1a7b570f9e\nStep 2/15 : LABEL author=\"Adam Uhlir <hello@adam-uhlir.me\"\n ---> Running in 241def94a6de\nRemoving intermediate container 241def94a6de\n ---> c15183e0671d\nStep 3/15 : LABEL description=\"Allows continuously publishing static pages from Git repository to IPFS\"\n ---> Running in 2462cf1422b6\nRemoving intermediate container 2462cf1422b6\n ---> 82e78631ac69\nStep 4/15 : ARG IPFS_PUBLISH_CONFIG=/data/ipfs_publish/config.toml\n ---> Running in cab29a205c6c\nRemoving intermediate container cab29a205c6c\n ---> 0476714cd522\nStep 5/15 : RUN apt-get -y install git   && mkdir -p /data   && adduser --home /data --uid 1000 --disabled-password --ingroup users ipfs_publish   && chown ipfs_publish:users /data\n ---> Running in 077eafe7328d\nReading package lists...\nBuilding dependency tree...\nReading state information...\ngit is already the newest version (1:2.30.2-1).\n0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.\nWarning: The home dir /data you specified already exists.\nAdding user `ipfs_publish' ...\nAdding new user `ipfs_publish' (1000) with group `users' ...\nThe home directory `/data' already exists.  Not copying from `/etc/skel'.\n\u001b[91madduser: Warning: The home directory `/data' does not belong to the user you are currently creating.\n\u001b[0mChanging the user information for ipfs_publish\nEnter the new value, or press ENTER for the default\n\tFull Name []: \tRoom Number []: \tWork Phone []: \tHome Phone []: \tOther []: \u001b[91mUse of uninitialized value $answer in chop at /usr/sbin/adduser line 582.\nUse of uninitialized value $answer in pattern match (m//) at /usr/sbin/adduser line 583.\n\u001b[0mIs the information correct? [Y/n] Removing intermediate container 077eafe7328d\n ---> 30a87e9e0e8c\nStep 6/15 : COPY ./startup.sh /usr/bin/ipfs-publish\n ---> c949b1c3a9c2\nStep 7/15 : USER ipfs_publish\n ---> Running in 5cf1dc007c9b\nRemoving intermediate container 5cf1dc007c9b\n ---> b08eead17ea9\nStep 8/15 : WORKDIR /app\n ---> Running in c5a68a3d72c0\nRemoving intermediate container c5a68a3d72c0\n ---> b7f22b4bd7a6\nStep 9/15 : RUN mkdir -p /data/ipfs_publish   && echo 'host = \"localhost\"\\nport = 8000\\n\\n[repos]\\n' > $IPFS_PUBLISH_CONFIG\n ---> Running in 8170b83aa4c7\nRemoving intermediate container 8170b83aa4c7\n ---> fecac25bcf96\nStep 10/15 : COPY . /app\n ---> b5342d491b4f\nStep 11/15 : RUN pip install --user .\n ---> Running in 54fd5b9b8f09\nProcessing /app\n  Preparing metadata (setup.py): started\n  Preparing metadata (setup.py): finished with status 'error'\n\u001b[91m  error: subprocess-exited-with-error\n  \n  × python setup.py egg_info did not run successfully.\n  │ exit code: 1\n  ╰─> [32 lines of output]\n      /usr/local/lib/python3.7/site-packages/setuptools/dist.py:700: UserWarning: Usage of dash-separated 'description-file' will not be supported in future versions. Please use the underscore name 'description_file' instead\n        % (opt, underscore_opt))\n      /usr/local/lib/python3.7/site-packages/setuptools/dist.py:700: UserWarning: Usage of dash-separated 'description-content-type' will not be supported in future versions. Please use the underscore name 'description_content_type' instead\n        % (opt, underscore_opt))\n      /usr/local/lib/python3.7/site-packages/setuptools/dist.py:700: UserWarning: Usage of dash-separated 'author-email' will not be supported in future versions. Please use the underscore name 'author_email' instead\n        % (opt, underscore_opt))\n      /usr/local/lib/python3.7/site-packages/setuptools/dist.py:700: UserWarning: Usage of dash-separated 'home-page' will not be supported in future versions. Please use the underscore name 'home_page' instead\n        % (opt, underscore_opt))\n      Traceback (most recent call last):\n        File \"<string>\", line 36, in <module>\n        File \"<pip-setuptools-caller>\", line 34, in <module>\n        File \"/app/setup.py\", line 6, in <module>\n          pbr=True,\n        File \"/usr/local/lib/python3.7/site-packages/setuptools/__init__.py\", line 152, in setup\n          _install_setup_requires(attrs)\n        File \"/usr/local/lib/python3.7/site-packages/setuptools/__init__.py\", line 147, in _install_setup_requires\n          dist.fetch_build_eggs(dist.setup_requires)\n        File \"/usr/local/lib/python3.7/site-packages/setuptools/dist.py\", line 788, in fetch_build_eggs\n          replace_conflicting=True,\n        File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 768, in resolve\n          replace_conflicting=replace_conflicting\n        File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 1051, in best_match\n          return self.obtain(req, installer)\n        File \"/usr/local/lib/python3.7/site-packages/pkg_resources/__init__.py\", line 1063, in obtain\n          return installer(requirement)\n        File \"/usr/local/lib/python3.7/site-packages/setuptools/dist.py\", line 844, in fetch_build_egg\n          return fetch_build_egg(self, req)\n        File \"/usr/local/lib/python3.7/site-packages/setuptools/installer.py\", line 52, in fetch_build_egg\n          eggs_dir = os.path.realpath(dist.get_egg_cache_dir())\n        File \"/usr/local/lib/python3.7/site-packages/setuptools/dist.py\", line 829, in get_egg_cache_dir\n          os.mkdir(egg_cache_dir)\n      PermissionError: [Errno 13] Permission denied: './.eggs'\n      [end of output]\n  \n  note: This error originates from a subprocess, and is likely not a problem with pip.\n\u001b[0m\u001b[91merror: metadata-generation-failed\n\n× Encountered error while generating package metadata.\n╰─> See above for output.\n\nnote: This is an issue with the package mentioned above, not pip.\nhint: See above for details.\n\u001b[0m\u001b[91mWARNING: You are using pip version 22.0.4; however, version 22.3.1 is available.\nYou should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.\n\u001b[0mRemoving intermediate container 54fd5b9b8f09\n","stderr":"The command '/bin/sh -c pip install --user .' returned a non-zero code: 1\n"}}